content/sensu-go/5.17/reference/entities.md

```diff
@@ -126,6 +129,8 @@ spec:
 
 {{< /language-toggle >}}
 
+_**NOTE**: The proxy entity definition must include the same subscriptions as the sensu-agent to work with round robin scheduling **and** response filter labels. If more than one sensu-agent will execute a proxy check and you did not configure the proxy entity with the same subscriptions as the sensu-agent, the proxy check will fail to run._
```

Comment:
```suggestion
_**NOTE**: The proxy entity definition must include the same subscriptions as the sensu-agent to work with round robin scheduling **and** [proxy requests attributes][24] . If more than one sensu-agent will execute a proxy check and you did not configure the proxy entity with the same subscriptions as the sensu-agent, an error will be logged by sensu-backend the proxy check will not be scheduled for agents to run._
```

Github won't let me add a suggestion for the link at the bottom, but I am thinking it would be `../checks/#proxy-requests-attributes`